packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -283,6 +294,10 @@ - (instancetype)initWithPlayerItem:(AVPlayerItem *)item
                                         error:nil] == AVKeyValueStatusLoaded) {
             // Rotate the video by using a videoComposition and the preferredTransform
             self->_preferredTransform = FVPGetStandardizedTransformForTrack(videoTrack);
+            // do not use video composition when it is not needed
+            if (CGAffineTransformIsIdentity(self->_preferredTransform)) {
```

Comment:
How is this change related to the PR? I don't see it discussed in the PR description.